common/lib/capa/capa/templates/clarification.html

```diff
@@ -0,0 +1,4 @@
+<span class="clarification">
```

Comment:
Also, as currently implemented, a screen reader user would be forced to consume this content (better than no access at all!) A nice touch would be to add a ```role="note" aria-label="Clarification"``` to this ```span``` which would give supporting screen reader  uses the option to consume the "Clarification note" or not.